src/Files.App/Actions/Navigation/OpenInNewTab/OpenInNewTabFromHomeAction.cs

```diff
@@ -6,6 +6,7 @@ namespace Files.App.Actions
 	internal sealed class OpenInNewTabFromHomeAction : BaseOpenInNewTabAction
 	{
 		public override bool IsExecutable =>
+			UserSettingsService.GeneralSettingsService.ShowOpenInNewTab &&
```

Comment:
Since `A && B` takes precedence over `A ? B : C`, parentheses should be used to prioritize `A ? B : C`. Otherwise,  IsExecutable will be true if ShowOpenInNewTab is false.

Reply:
Done